data/virt_autotest/VIRT_TEST_VM_config.ign

```diff
@@ -0,0 +1,25 @@
+{
+  "ignition": {
+    "version": "3.0.0"
+  },
+  "passwd": {
+    "users": [
+      {
+        "name": "root",
+        "passwordHash": "$1$YRC3ONRz$6d4zi/2z5Eq72tuFN/cVR0"
+      }
+    ]
+  },
+  "storage": {
+    "files": [
+      {
+        "path": "/etc/ssh/sshd_config",
+        "mode": 420,
+        "overwrite": true,
+        "contents": {
+          "source": "data:text/plain;charset=utf-8;base64,UGVybWl0Um9vdExvZ2luIHllcwpQdWJrZXlBdXRoZW50aWNhdGlvbiB5ZXMKUGFzc3dvcmRBdXRoZW50aWNhdGlvbiB5ZXM="
+        }
+      }
+    ]
+  }
+}
```

Comment:
I think this config file can be moved to data/virt_autotest/guest_unattended_installation_files and rename it to alp_xxxx_on_alp_xxxxx or alp_xxxx_on_other_hosts_xxxx.

Reply:
It will be the same for different hosts, but will move it to data/virt_autotest/guest_unattended_installation_files.

Reply:
done